nexus-config/src/nexus_config.rs

```diff
@@ -441,6 +441,8 @@ pub struct BackgroundTaskConfig {
     pub webhook_deliverator: WebhookDeliveratorConfig,
     /// configuration for SP ereport ingester task
     pub sp_ereport_ingester: SpEreportIngesterConfig,
+    /// reconfigurator runtime configuration
+    pub chicken_switches: ChickenSwitchesConfig,
```

Comment:
Small nit - should these live under `BlueprintTasksConfig`? So in the TOML, it'd be something like

```toml
blueprint.chicken_switches.loader_period_secs = N
```

or if we only expect this to ever really have the period, maybe eliminate the `ChickenSwitchesConfig` struct and make it

```toml
blueprints.chicken_switches_loader_period_secs = N
```

Reply:
Good call. Done in [03cda10](https://github.com/oxidecomputer/omicron/pull/8462/commits/03cda10b498566b5b7ac7910eb34f32c4139bde9)